dev/io.openliberty.webcontainer.servlet.6.0.internal_fat/test-applications/SimpleTest.war/src/simpletest/servlets/TestServlet.java

```diff
@@ -12,6 +12,7 @@
 
```

Comment:
Are you planning to update the tests to verify anything in this servlet in this PR or plan to do that at a later date?

Reply:
It still needs to be updated.  Right now all these APIs output are showing up in the log.

Reply:
Can you get an issue opened then so we don't forget to write the tests for this? Just link in this issue so we can track it. Thanks!

Reply:
tracking under https://github.com/OpenLiberty/open-liberty/issues/20425#issue-1168753818